src/systemtask/SystemTask.cpp

```diff
@@ -351,6 +363,9 @@ void SystemTask::Work() {
       batteryNotificationTick = xTaskGetTickCount();
     }
 
+    
+    console.Process();
+
```

Comment:
Is the accelerometer values reporting the only reason this is called periodically? Is there a better way to do it?

Reply:
`Process()` was primarily added so the execution of the buffered command (and stack) is in the systemtask context and not the nimble callback context.
Then it was handy for accelerometer values cause it is called in a reasonable ~10Hz pace?

Reply:
You could move it to a SystemTask Message so it only gets run when needed, but the accelerometer reporting would need another solution. The accelerometer part should probably be a separate function.

Reply:
Current commands are really to just show off the posibilities of console. If I do just the console then not all people might understand what it is good for. We can delete all the commands if that means faster merge with develop.

Reply:
I got it now :) You mean separate function, but within current Console class? I'll do that.

Reply:
Moved lvgl command and acceleromter debug to functions in f15ce01

Reply:
What about moving the Process() function to a new SystemTask Message? When a command is received, it could just push a message to systemtask to process it. The reason for separating AccelerometerDebug() was so it could be called from somewhere else than Process() since that is the only part that needs to be called periodically right?

Reply:
I don't feel that it shuld be event driven. I can imagine that we might need some  polling in case Console.Print() will have too long message to fit into a single BLE packet, then you need to buffer and divide that and this should be also handled in Console Process polling function.
There is not any overhead in Process, since its checking just a bool and exits immediatelly.

Reply:
Couldn't the function be made to not return before the entire message is sent, requiring only one call to Process() even if the message needs to be split? Process() primarily just reads the received command, so having to call it multiple times for a single command doesn't really make sense and I don't see why it shouldn't be event based. Maybe you're trying to do too many things in one function.

Reply:
Changed console parsing to the new event Messages::ConsoleProcess. e94d0fd
Removed accelerometer completely from this PR to focus on main functionality.